ecs-agent/daemonimages/csidriver/driver/node.go

```diff
@@ -154,19 +163,26 @@ func (d *nodeService) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 		if part != "0" {
 			partition = part
 		} else {
-			klog.InfoS("NodeStageVolume: invalid partition config, will ignore.", "partition", part)
+			klog.V(4).InfoS("NodeStageVolume: invalid partition config, will ignore.", "partition", part)
 		}
 	}
 
+	klog.V(4).Infof("NodeStageVolume: configs initialized for task %s and volume %s, time taken: %f seconds", taskId, volumeID, time.Since(nodeStageStartTime).Seconds())
+
+	findevicepathTime := time.Now()
```

Comment:
please use camel case for this and all time.Now() additions below.

Reply:
These have been removed in [502174e](https://github.com/aws/amazon-ecs-agent/pull/4326/commits/502174ee6c28610cf856718ec299d900f98eb65b)